source/faucet/main.d

```diff
@@ -41,7 +41,7 @@ import vibe.web.rest;
 private struct Config
 {
     /// How frequently we run our periodic task
-    static immutable interval = 15.seconds;
+    static immutable interval = 30.seconds;
```

Comment:
This should be moved into its own commit and explained why the change was made.

Reply:
Fixed